src/lexer/scanner.go

```diff
@@ -290,6 +305,11 @@ func (s *Scanner) Scan() Token {
 			s.dft.Reset()
 			s.file.Seek(-1, os.SEEK_CUR)
 
+			s.currentColumnFile -= n
```

Comment:
Here, if we want to provide any information regarding to the word itself, maybe we should subtract the length of lexem, no?

Reply:
I agree, but I think that there are more things to be considered regarding the error information when we are dealing with words. This bug fix tries to specifically deal with unknown characters occurrence.

Reply:
We can deal with words in a future update.